### PR TITLE
Fix docker image

### PR DESCRIPTION
**Pull request by goodtiding5**
Use external volume to keep user data persistent.

Place 'nim' executable at '/usr/local/bin'.

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+set -e
+set -u
+
+BUILD=/build
+WORKD=/data
+
+build_working_dir() {
+    [ -f $WORKD/nitter.conf ] || cp -f  $BUILD/nitter.conf $WORKD/.
+    [ -d $WORKD/public ]      || cp -rf $BUILD/public      $WORKD/.
+    [ -d $WORKD/tmp ]         || mkdir -p $WOKRD/tmp
```
Comment (setenforce0) [inline entrypoint.sh]: Typo
```suggestion
    [ -d $WORKD/tmp ]         || mkdir -p $WORKD/tmp
```

Comment (goodtiding5) [reply entrypoint.sh]: Good catch!  I will make the fix and generate a new merge request.

Comment (zedeus) [reply entrypoint.sh]: No need for a new request, just push the changes to your branch

Comment (goodtiding5) [reply entrypoint.sh]: OK.  Done!

**Comment by goodtiding5** [comment]
README file is updated.  Please review.

**Comment by Perflyst** [comment]
Cache is stored in redis, the tmp folder is gone. If you want persistent configuration you can simply mount nitter.conf. I think this can be closed @zedeus

**Pull request closed without merging**